src/share/dateRangePicker/DatePicker.tsx

```diff
@@ -0,0 +1,164 @@
+import React, {
```

Comment:
oh, I'll ignore this file if you don't mind. I have no idea why material ui datepicker needs so many code for setup

Reply:
it's error prone because the more code you write the bigger probability of errors

Reply:
Why you need all this code (this file and several files below)? wouldn't be better to use another component which supports DateRange out of the box without this terrible setup?

Reply:
I tried to use ready date range picker component based on material-ui but it didn't enough customizable